hack/verify-crds-kind.sh

```diff
@@ -108,39 +165,43 @@ for check in {1..10}; do
 done
 
 for CHANNEL in experimental standard; do
-  ##### Test v1alpha2 CRD apply and that invalid examples are invalid.
+  ##### Test valid CRD apply and that invalid examples are invalid.
   # Install CRDs
   kubectl apply -f "config/crd/${CHANNEL}/gateway*.yaml" || res=$?
 
   # Temporary workaround for https://github.com/kubernetes/kubernetes/issues/104090
   sleep 8
 
-  kubectl apply --recursive -f examples/standard || res=$?
+  # Note that we skip the working examples since we did them already with
+  # just CEL validation.
 
-  # Install all experimental example gateway-api resources when experimental mode is enabled
-  if [[ "${CHANNEL}" == "experimental" ]]; then
-    echo "Experimental mode enabled: deploying experimental examples"
-    kubectl apply --recursive -f examples/experimental || res=$?
-  fi
+  for file in $(find hack/invalid-examples -name "*.yaml"); do
+    # Don't check alpha resources in Standard checks
+    if [[ "$file" =~ "v1alpha2" && "$CHANNEL" == "standard" ]]; then
```

Comment:
Maybe we should just be using `hack/invalid-examples/experimental` instead of `hack/invalid-examples/v1alpha2` to be more consistent with what we're doing with other examples.

Reply:
Hmm, yeah, I can move the examples to `experimental` and `standard` directories instead of `v1alpha2` and `v1beta1` directories? That will be consistent with the working examples dir.